frontend/src/services/project.ts

```diff
@@ -31,3 +33,20 @@ export async function getProjects(): Promise<Project[]> {
     });
     return result.projects;
 }
+
+export async function getGroups(projectId: number): Promise<Group[]> {
+    const result = await authorized_fetch<{ groups: Group[] }>(
+        `/api/projects/${projectId}/groups`,
+        {
+            method: "GET",
+        }
+    );
+    return result.groups;
+}
+
+export async function getSubmissions(projectId: number): Promise<Submission[]> {
+    const result = await authorized_fetch<Submission[]>(`/api/projects/${projectId}/submissions`, {
+        method: "GET",
+    });
+    return result;
+}
```

Comment:
Om consequent te zijn met de rest van de query refactor zou ik deze dan in `services/submission.ts` steken en `getProjectSubmissions` noemen.